### ISSUE TITLE
Test for WCAG technique H48: part 2: Using dl for lists

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#154]
Opened this issue to investigate the requirements for detecting text that should be description lists (formerly known as definition lists) as part of #116 . As definition lists are rarely used, we first need some examples of code that should have been a description list.
